cmd/clusterctl/client/delete.go

```diff
@@ -53,6 +53,9 @@ type DeleteOptions struct {
 
 	// IncludeCRDs forces the deletion of the provider's CRDs (and of all the related objects).
 	IncludeCRDs bool
+
+	// IncludeInventory forces the deletion of the inventory items used by clusterctl to track providers.
+	IncludeInventory bool
```

Comment:
I think we should invert this flag and name it skipInventory, so the inventory gets deleted by default (false).
Otherwise we are breaking all the clients calling delete and relying on the current behaviour